spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 ENV["DB"] ||= "sqlite"
 
 require "byebug"
+require "support/database_specific_specs"
```

Comment:
Please create a separate PR for this. It's a good idea, but I'd like to handle it separately, and more comprehensively. For this PR, please use plain ruby conditionals the way other tests currently do.